src/core/Pattern_match.ml

```diff
@@ -153,6 +153,7 @@ and rule_id = {
    * mini_rule? *)
   message : string;
   fix : string option;
```

Comment:
I think we should fix the TODO above at some point (not in this PR though) and remove
all those fields from rule_id